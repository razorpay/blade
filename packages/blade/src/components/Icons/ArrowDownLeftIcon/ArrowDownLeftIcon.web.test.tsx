import ArrowDownLeftIcon from './';
import renderWithTheme from '~utils/testing/renderWithTheme.web';

describe('<ArrowDownLeftIcon />', () => {
  it('should render ArrowDownLeftIcon', () => {
    const { container } = renderWithTheme(
      <ArrowDownLeftIcon color="feedback.icon.neutral.intense" size="large" />,
    );
    expect(container).toMatchSnapshot();
  });
});
