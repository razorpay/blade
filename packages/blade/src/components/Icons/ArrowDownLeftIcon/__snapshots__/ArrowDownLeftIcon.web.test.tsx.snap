// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ArrowDownLeftIcon /> should render ArrowDownLeftIcon 1`] = `
<div>
  <svg
    aria-hidden="true"
    data-blade-component="icon"
    fill="none"
    height="20px"
    viewBox="0 0 24 24"
    width="20px"
  >
    <path
      d="M18.7071 6.70711C19.0976 6.31658 19.0976 5.68342 18.7071 5.29289C18.3166 4.90237 17.6834 4.90237 17.2929 5.29289L7 15.5858V9C7 8.44772 6.55228 8 6 8C5.44772 8 5 8.44772 5 9V17.9998C5 18.1354 5.02699 18.2649 5.07588 18.3828C5.12432 18.4999 5.19595 18.6096 5.29078 18.705C5.29219 18.7064 5.2936 18.7078 5.29502 18.7092C5.48924 18.9023 5.74301 18.9992 5.997 19C5.998 19 5.999 19 6 19H15C15.5523 19 16 18.5523 16 18C16 17.4477 15.5523 17 15 17H8.41421L18.7071 6.70711Z"
      fill="hsla(216, 27%, 36%, 1)"
    />
  </svg>
</div>
`;
